dataikuapi/dss/recipe.py

```diff
@@ -1376,6 +1376,37 @@ def with_output_evaluation_store(self, mes_id):
         return self._with_output(mes_id, role="evaluationStore")
 
 
+class StandaloneEvaluationRecipeCreator(DSSRecipeCreator):
+    """
+    Builder for the creation of a new "Standalone Evaluate" recipe, from an
+    input dataset
+
+    .. code-block:: python
+
+        # Create a new standalone evaluation of a dataset
+
+        project = client.get_project("MYPROJECT")
+        builder = StandaloneEvaluationRecipeCreator("my_standalone_evaluation_recipe", project)
+        builder.with_input("scored_dataset_to_evaluate")
+        builder.with_output_evaluation_store(evaluation_store_id)
+
+        new_recipe = builder.build()
+
+    Output model evaluation store must exist. It can be created using the following:
+
+    .. code-block:: python
+
+        evaluation_store_id = project.create_model_evaluation_store("output_model_evaluation").mes_id
+    """
```

Comment:
It would be nice to add to the documentation the payload parameters, such as `isProbaAware`, `dontComputePerformance`, etc.
Would also be great to do the same for the evaluation recipe.
It was not already done for this last, and your were right to do something similar for Standalone Evaluation Recipe, but this would be useful, IMHO.

Reply:
I agree